schemas/compat-data-schema.md

```diff
@@ -369,6 +369,19 @@ Example for two flags required:
 }
 ```
 
+#### `implementation_url`
+
+An optional changeset URL or commit URL for the revision that implemented the feature in the
```

Comment:
A few things here:

1. What do you think of breaking this up visually?
2. Based on the discussion, can we drop "spec-conformant" bit? I think hashing out "is this URL good enough for `implementation_url`?" is probably better suited to the data guidelines; consumers ought to be able to trust that we've done some adequate vetting that the URL demonstrates support without needing to know the gritty details.
3. Wordsmithing the "even if the feature hasn't shipped" stuff, particularly to make "Safari Technology Preview" more searchable.

Here's sorta what I had in mind (open to modification/reinterpretation):

> An optional URL that links to the commit or changeset for the revision that implemented the feature in the source code for the browser or underlying engine.
> 
> For example, a URL from:
> 
> - `https://trac.webkit.org/changeset/`
> - `https://hg.mozilla.org/mozilla-central/rev/`
> - `https://crrev.com/`
> 
> Typically, an applicable change has a subject line in the form of _"Implement [feature]"_,
_"Support [feature]"_, or _"Enable [feature]"_.
>
> The presence of an `implementation_url` value indicates that the associated browser has support for the feature in its source code. Therefore, an `implementation_url` value can be added even if the feature
hasn't yet shipped in a stable version, but may be available in a development version (such as Firefox Nightly, Chrome Canary, or Safari Technology Preview).